### PR TITLE
LPS-118424 Fix Field ID label on Rule Builder

### PR DESCRIPTION
**Pull request by brunofariasdeo**
https://issues.liferay.com/browse/LPS-118424

### CODE REVIEW

**Comment by brunofariasdeo** [comment]
ci:test:sf

**Comment by brunofariasdeo** [comment]
ci:test:relevant

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/components/Calculator/Calculator.soy [hunk 1]
```diff
@@ -134,7 +134,7 @@
 
 	{let $item kind="html"}
 		{$label}
-		<span class="calculate-fieldname">Fieldname: {$title}</span>
+		<span class="calculate-fieldname">{msg desc=""}field-id{/msg}: {$title}</span>
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/components/Calculator/Calculator.soy:137]: @brunofariasdeo, your fix seems good to me, but can you please give me more details about `{msg ...}`?

Thanks.

**Comment by natocesarrego** [comment]
ci:forward

**Pull request closed without merging**